database/buildSchema/19_b_application_stage_status.sql

```diff
@@ -0,0 +1,43 @@
+-- Create VIEW which collects application, stage and status information together
+CREATE VIEW public.application_stage_status AS
+	(SELECT app.id,
+		name,
+		ts.number AS stage_number,
+		ts.title AS stage,
+		status
+	FROM application app
+	JOIN application_stage_history stage ON app.id = stage.application_id
+	JOIN template_stage ts ON stage.stage_id = ts.id
+	JOIN application_status_history status ON stage.id = status.application_stage_history_id
+	WHERE stage.is_current = TRUE
+	AND status.is_current = TRUE );
```

Comment:
I think `ts` is too hard to understand in a glimpse. Can this be changed to `stages` maybe?

Reply:
Or `temp_stage`

Reply:
Sure, I was just following the convention I've noticed in a lot SQL where aliases are super short and basically initials -- I guess it keeps the typing down? 

Anyway, fine to just lose the alias altogether? (Done in 65c9a41)